he.lproj/Localizable.strings

```diff
@@ -161,73 +161,73 @@
+"EDIT_STATION_WINDOW_INTERNET_RADIO_PLACEHOLDER" = "רדיו אינטרנט";
 
-"STATION_METADATA_NAME_PLACEHOLDER" = "Untitled Station";
+"STATION_METADATA_NAME_PLACEHOLDER" = "תחנה חדשה";
 
-"WELCOME_BUTTON_CONTINUE" = "Continue";
-"WELCOME_BUTTON_GETSTARTED" = "Get Started";
+"WELCOME_BUTTON_CONTINUE" = "המשך";
+"WELCOME_BUTTON_GETSTARTED" = "התחל";
 
-"WELCOME_NEWFEATURES_TITLE" = "New Features";
+"WELCOME_NEWFEATURES_TITLE" = "תכונות חדשות";
 
-"WELCOME_NEWFEATURES_2_DESIGN" = "Modern look & feel";
-"WELCOME_NEWFEATURES_2_SLEEPTIMER" = "Sleep Timer";
-"WELCOME_NEWFEATURES_2_AIRPLAY" = "AirPlay toolbar controls";
-"WELCOME_NEWFEATURES_2_ALERTS" = "Less-intrusive alerts";
-"WELCOME_NEWFEATURES_2_NATIVESCALING" = "Native pixel scaling";
-"WELCOME_NEWFEATURES_2_APPLESILICON" = "Supports Apple Silicon";
+"WELCOME_NEWFEATURES_2_DESIGN" = "מראה ותחושה מודרניים";
+"WELCOME_NEWFEATURES_2_SLEEPTIMER" = "שעון עצר לשינה";
+"WELCOME_NEWFEATURES_2_AIRPLAY" = "שליטה על AirPlay";
+"WELCOME_NEWFEATURES_2_ALERTS" = "התראות פחות מפריעות";
+"WELCOME_NEWFEATURES_2_NATIVESCALING" = "תמיכה ברזולוציות מסך גבוהות";
+"WELCOME_NEWFEATURES_2_APPLESILICON" = "תמיכה ב- Apple Silicon";
 
-"MENU_CONTROLS" = "Controls";
-"MENU_CONTROLS_PLAY" = "Play";
-"MENU_CONTROLS_PAUSE" = "Pause";
-"MENU_CONTROLS_NEXT" = "Next";
-"MENU_CONTROLS_PREVIOUS" = "Previous";
+"MENU_CONTROLS" = "שליטה";
+"MENU_CONTROLS_PLAY" = "נגן";
+"MENU_CONTROLS_PAUSE" = "השהה";
+"MENU_CONTROLS_NEXT" = "הבא";
+"MENU_CONTROLS_PREVIOUS" = "הקודם";
 
-"SLEEP_TIMER_MENU" = "Sleep Timer";
+"SLEEP_TIMER_MENU" = "שעון עצר לשינה";
 
-"SLEEP_TIMER_MENU_DISABLED" = "Disabled";
-"SLEEP_TIMER_MENU_1" = "1 Minute";
-"SLEEP_TIMER_MENU_15" = "15 Minutes";
-"SLEEP_TIMER_MENU_30" = "30 Minutes";
-"SLEEP_TIMER_MENU_60" = "60 Minutes";
+"SLEEP_TIMER_MENU_DISABLED" = "לא פעיל";
+"SLEEP_TIMER_MENU_1" = "1 דקה";
+"SLEEP_TIMER_MENU_15" = "15 דקות";
+"SLEEP_TIMER_MENU_30" = "30 דקות";
+"SLEEP_TIMER_MENU_60" = "60 דקות";
 
-"EDIT_STATION_WINDOW_NAME_FIELD_LABEL" = "name";
-"EDIT_STATION_WINDOW_URL_FIELD_LABEL" = "address";
-"EDIT_STATION_WINDOW_IMAGE_FIELD_LABEL" = "artwork";
+"EDIT_STATION_WINDOW_NAME_FIELD_LABEL" = "שם";
+"EDIT_STATION_WINDOW_URL_FIELD_LABEL" = "כתובת";
+"EDIT_STATION_WINDOW_IMAGE_FIELD_LABEL" = "עטיפה";
 
-"EDIT_STATION_WINDOW_IMAGE_BUTTON_CHOOSE" = "Choose Image…";
-"EDIT_STATION_WINDOW_LIMIT_REACHED" = "Trial library limit reached. Please upgrade to add more stations.";
+"EDIT_STATION_WINDOW_IMAGE_BUTTON_CHOOSE" = "בחר תמונה...";
+"EDIT_STATION_WINDOW_LIMIT_REACHED" = "הגעת לתקרת ספריית הנסיון. שדרג כדי להוסיף תחנות נוספות.";
 
-"TOOLBAR_PREVIOUSBUTTON_TOOLTIP" = "Previous";
-"TOOLBAR_NEXTBUTTON_TOOLTIP" = "Next";
+"TOOLBAR_PREVIOUSBUTTON_TOOLTIP" = "הקודם";
+"TOOLBAR_NEXTBUTTON_TOOLTIP" = "הבא";
 
-"ALERT_UPGRADE" = "Upgrade";
+"ALERT_UPGRADE" = "שדרג";
 
-"MENU_APPLICATION_UPGRADE" = "Upgrade to Full Version…";
+"MENU_APPLICATION_UPGRADE" = "שדרוג לגירסא המלאה...";
```

Comment:
What do we use with all the other verbs? It should all either be "עיון", "שדרוג", "ניגון", or "עיין", "שדרג", "נגן", but not both.